docs/proposals/multi-node/README.md

```diff
@@ -581,6 +586,13 @@ The `members` section of the status is intended to be updated by the `etcd-backu
 Especially, members can be marked with `status: Ready` only by their `etcd-backup-restore` sidecar container.
 However, they can be marked with `status: NotReady` either by their `etcd-backup-restore` sidecar container (with `reason: HeartbeatFailed`) or by `etcd-druid` (as explained [below](#decision-table-for-etcd-druid-based-on-the-status)).
 
+In an ETCD cluster, the member `id` is the [unique identifier for a member](https://etcd.io/docs/v3.4/dev-guide/api_reference_v3/#message-member-etcdserveretcdserverpbrpcproto).
+However, this proposal recommends using a [single `StatefulSet`](#kubernetes-context) whose pods form the members of the ETCD cluster and `Pods` of a `StatefulSet` have [uniquely indexed names](https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/#ordinal-index) as well as [uniquely addressible DNS](https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/#stable-network-id).
+
+This proposal recommends that the `name` of the member (which is the same as the name of the member `Pod`) be used as the unique key to identify a member in the `members` array.
+This can minimise the need to cleanup superfluous entries in the `members` array after the member pods are gone to some extent because the replacement pods for any member will share the same `name` and will overwrite the entry with a [possibly new](#restarting-an-existing-member-of-an-etcd-cluster) member `id`. 
+There is still the possibility of not only [superfluous entries in the `members` array](#13-superfluous-member-entries-in-etcd-status) but also [superfluous `members` in the ETCD cluster](#work-flows-only-on-the-leading-member) for which there is no corresponding pod in the `StatefulSet` anymore both of which need to be [cleaned up](#recommended-action-12) [as appropriate](#work-flows-only-on-the-leading-member).
```

Comment:
What do you think about giving an example here:
Cluster is scaled up `3 -> 5`, two new members are added but then constantly crash for OOM reasons. Then cluster is scaled down `5 -> 3` which deletes pods `etcd-main-4`, `etcd-main-5` but `len(status.members) == 5` and `members == 5` in the cluster.

Reply:
👍 Will do